Chapters/01-zig-weird.qmd

```diff
@@ -205,13 +219,15 @@ The function returns an integer of the type `i32` as result.
 Zig is not exactly a strongly-typed language. Because you can (if you want to) omit
 the type of an object in your code, if this type can be derived from the assigned value.
 But there are other situations where you do need to be explicit.
-For example, you do have to explicitly specify the type of every single function argument, and also,
+For example, you do have to explicitly specify the type of every 
+<!-- 'each function argument' instead? -->

```

Comment:
I think this boils down to preference. But, I'm ok with using "each function argument" here. It works anyway.